engine/utils/ocaml_of_json_schema/ocaml_of_json_schema.js

```diff
@@ -208,6 +220,10 @@ let parse_type_name = s => {
     return s.split('/').slice(-1)[0];
 };
 
+let int_repr_of_format = format => 
+  (format.endsWith('int128') || format == 'uint64' || format == 'uint' /*`uint`s are `usize`s actually, so that's safer to assume it's a uint64, see https://github.com/GREsau/schemars/blob/386e3d7f5ac601795fb4e247291bbef31512ded3/schemars/src/json_schema_impls/primitives.rs#L85C16-L85C21*/)
```

Comment:
🙈

Reply:
Ahaha, yes, that is really terrible... That caused me quite a lot of pain